packages/venia-concept/src/components/MiniCart/cartOptions.css

```diff
@@ -61,6 +61,42 @@
     composes: title from '../../components/ProductOptions/option.css';
 }
 
+.quantity button {
+    height: 36px;
+    background-color: #ccc;
+    border: 0;
+    border-radius: 4px;
+    width: 36px;
+    font-size: 28px;
+    line-height: 30px;
+    padding: 0px 0 6px 0;
+    vertical-align: top;
+}
+.quantity button:hover {
+    background-color: #333;
+    color: #fff;
+}
+.qtyfield {
```

Comment:
Should not abbreviate here. Suggest `quantityInput` or simply `input`.

Reply:
@brendanfalkowski :
Have resolved the same in latest commit.